### PR TITLE
Fix: Change CodeEditor placeholder color

### PR DESCRIPTION
**Pull request by ergilaze**

### CODE REVIEW

File: src/components/CodeEditor/index.jsx [hunk 1]
```diff
@@ -16,6 +16,9 @@ import './json-lint';
 @withStyles({
   root: {
     width: '100%',
+    '& .CodeMirror-placeholder': {
+      color: '#656469 !important',
+    },
```
Comment (helfi92) [inline src/components/CodeEditor/index.jsx]:
```diff
   root: {
     width: '100%',
     '& .CodeMirror-placeholder': {
-      color: '#656469 !important',
+      // Same as demo
+      // https://github.com/codemirror/CodeMirror/blob/e3b9957f3e46aad5c0c2fd309f65b6fec10d267d/demo/placeholder.html#L14
+      color: '#999',
     },
   },
 })
```

Was there a reason for adding `!important`?

Comment (ergilaze) [reply src/components/CodeEditor/index.jsx]: I tried without `!important`, but the color didn't change.

**Comment by ergilaze** [comment]
@helfi92 Super :) Tell me if there is any other issue you want me to have a look at ...

**Comment by helfi92** [comment]
I just merged the update in the taskcluster repo but it seems like there's something overwriting that change so the color of the placeholder is not showing. Do you want to have a look? You can access the editor with something like http://localhost:5080/secrets/create

**Comment by helfi92** [comment]
I suspect the change will need to be done in the taskcuster repo.

**Comment by ergilaze** [comment]
@helfi92 I will have a look at it later.

**Pull request merged**